04/2_print_string_buffer/buffered_print.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <string.h>
+
+#include "buffered_print.h"
+
+#define BUFFER_LENGTH (16)
+
+static int s_buffer_chars = 0;
+static char s_buffer[BUFFER_LENGTH];
+
+void buffered_print(const char* src)

```

Comment:
not implemented to the spec.

you don't seem to understand what this function should do

does this even work with the cases that I gave you in the last review?

Reply:
without a loop you cannot implement this function ever